### DISCUSSION TITLE
remove Edge telemetry without disabling DNS?

### DISCUSSION THREAD

**Discussion by mncorrea** [ChrisTitusTech/winutil#827] [Q&A]
al eliminar la telemetría del navegador Edge, impide establecer un DNS...

Hi. I am writing to ask if it is possible to remove Edge telemetry without disabling the DNS option in the browser settings. I want to configure a specific DNS for the browser and it won't let me. I deleted the policies folder in the registry editor and configured the browser manually, but I was left wondering if that was possible to implement. could you help me with that? Thanks.
Greetings from Argentina.

in the image, I expose the configuration in the browser I refer to.
![Captura de pantalla 2023-06-07 092538](https://github.com/ChrisTitusTech/winutil/assets/121661789/3744ef4a-efae-4009-a1af-8fa613fcee7f)

**Comment by cdock1029** [comment]
I also would like to Use Secure DNS. Is there any way to turn this on with Edge Debloat still active?

**Comment by xlionjuan** [comment]
Add `DnsOverHttpsTemplates` still not works

https://learn.microsoft.com/en-us/deployedge/microsoft-edge-browser-policies/dnsoverhttpstemplates
